src/Addon/Assets.php

```diff
@@ -0,0 +1,52 @@
+<?php
+namespace GiveAddon\Addon;
+
+/**
+ * Helper class responsible for loading add-on assets.
+ *
+ * @package     GiveAddon\Addon
+ * @copyright   Copyright (c) 2020, GiveWP
+ */
+class Assets {
+
+	/**
+	 * Load add-on backend assets.
+	 *
+	 */
+	public static function loadBackendAssets() {
+		wp_enqueue_style(
+			'give-addon-boilerplate-style-backend',
+			ADDON_CONSTANT_URL . 'public/css/give-addon-admin.css',
+			[],
+			GIVE_VERSION
+		);
+
+		wp_enqueue_script(
+			'give-addon-boilerplate-script-backend',
+			ADDON_CONSTANT_URL . 'public/js/give-addon-admin.js',
+			[],
+			GIVE_VERSION,
+			true
+		);
+	}
+
+	/**
+	 * Load add-on front-end assets.
+	 */
+	public static function loadFrontendAssets() {
+		wp_enqueue_style(
```

Comment:
I think we can set give style and script as a dependency for frontend script. 

@JasonTheAdams What do you think?

Reply:
No need for dependencies, the boilerplate add-on won't even load if the GiveWP is not activated.

Reply:
@ravinderk It really depends on the script — whether or not it really does need to load before or after a Give script. There isn't a one-size-fits-all solution to this.

@alaca Whether or not this loads if GiveWP is installed isn't the point; it affects the order in which scripts are appended to the outgoing page.